app/models/relaymeter/relaymeter.data.ts

```diff
@@ -1,13 +1,16 @@
 import { RelayMetric } from "./relaymeter.server"
+import { AccountAppRelays } from "~/routes/account.$accountId.settings.plan/route"
 
-export const relayMetric: RelayMetric = {
+export const relayMetric: AccountAppRelays = {
```

Comment:
why is this type changing

Reply:
Mock data for the test. It changed from a single RelayMetric to an array of RelayMetric per app under the account.